package.json

```diff
@@ -15,7 +15,7 @@
     "build": "turbo run build --scope=next-auth --scope=@next-auth/* --no-deps",
     "lint": "turbo run lint --scope=!next-auth-docs",
     "lint:fix": "turbo run lint -- --fix",
-    "test": "turbo run test --concurrency=1 --scope=!@next-auth/pouchdb-adapter --scope=!next-auth-*",
+    "test": "turbo run test --concurrency=1 --scope=!@next-auth/pouchdb-adapter --scope=!next-auth-* --filter=[HEAD^1]",
```

Comment:
Since --scope is deprecated in 1.2.x, do you think we should remove all `--scope` in this PR too?

Reply:
Where did you see that? Not finding anything in their [changelog](https://github.com/vercel/turborepo/releases) :thinking: 

But yeah in that case lets remove it and/or find a replacement for it. The point of the two `--scope` arguments (as far as I understood it) was the following:

- To ignore v3 adapters (i.e. `pouchdb-adapter`)
- To not build when changes are only in `next-auth-app` or `next-auth-gatsby`, i.e. those kinds of demo apps/directories.

Reply:
Yeah, sorry I didn't link it (was on mobile 😄) https://turborepo.org/docs/core-concepts/scopes It's on their docs site
> The point of the two --scope arguments (as far as I understood it) was the following:

    To ignore v3 adapters (i.e. pouchdb-adapter)
    To not build when changes are only in next-auth-app or next-auth-gatsby, i.e. those kinds of demo apps/directories.

You got it right!